### ISSUE TITLE
RCTRootView cancelTouches]` is deprecated and will be deleted soon.

### ISSUE THREAD

**Issue by darvinueza** [facebook/react-native#26626]
React Native version:

## Steps To Reproduce

1. I am using Import MapView, {Marker} from "react-native-maps"; and I have this error when clicking on the map: 
-[RCTRootView cancelTouches]` is deprecated and will be deleted soon.
2.

Describe what you expected to happen:

Snack, code example, screenshot, or link to a repository:

**Comment by Kamran1122** [comment]
same issue

**Comment by Zontex** [comment]
Same here

**Comment by jonstuebe** [comment]
see https://github.com/kmagiera/react-native-gesture-handler/pull/657

**Comment by m4rcoperuano** [comment]
This causes crashes when you build for Release

**Comment by m4rcoperuano** [comment]
I can confirm that adding this line of code as the _first_ line in my ~/index.js has solved the problem. Weird.

`import 'react-native-gesture-handler';`

**Comment by mackenziemance** [comment]
This seems to be causing a crash in debug when focusing on inputs

**Comment by kamilzamerski** [comment]
The same... it freeze app when focus on input....

**Comment by Kamran1122** [comment]
Any fix for this yet?

**Comment by mackenziemance** [comment]
Reporting same behaviour as @kamilzamerski

**Comment by mackenziemance** [comment]
#26707

**Comment by gabrielrra** [comment]
> I can confirm that adding this line of code as the _first_ line in my ~/index.js has solved the problem. Weird.
> 
> `import 'react-native-gesture-handler';`

This solved the warning for me

**Comment by igor-lemon** [comment]
> import 'react-native-gesture-handler';

Doesn't work for me... :(
Any ideas?

**Comment by Kamran1122** [comment]
> `import 'react-native-gesture-handler';`
doesn't work for me either

**Comment by luism3861** [comment]
same issue

**Comment by wesleynfreitas** [comment]
I try put

1 - import 'react-native-gesture-handler' at the top of index.js
2 - YellowBox.ignoreWarnings([
    'RCTRootView cancelTouches'
]);

And don't works. Any news about the solution?

**Comment by alesdvorakcz** [comment]
I have same problem. In dev I see warnings, in production build it crashes in ios. Looks like many community packages relies on this. (react-gesture-handler, react-native-maps, @react-native-community/datetimepicker, ...)

**Comment by taminhtien** [comment]
Same problem for me

```
    "react": "16.8.1",
    "react-native": "0.61.2",
    "react-native-reanimated": "^1.3.0",
    "react-native-screens": "^1.0.0-alpha.23",
    "react-navigation": "^4.0.10",
    "react-navigation-stack": "^1.10.2"
```

**Comment by osamaq** [comment]
Causes crash in release builds. For android, I could troubleshoot it by importing the gesture handler at the top of `index.js`. For iOS, haven't found a solution yet...

**Comment by mauricioblum** [comment]
+1 here. Crashing iOS release builds - RN version 0.61.2

**Comment by JuanCamilo0809** [comment]
> +1 here. Crashing iOS release builds - RN version 0.61.2

same problem

**Comment by mxhryvo** [comment]
Same for me

**Comment by YutaSugimura** [comment]
Recommendation: downgrade react-native

rewrite package.json
``` package.json
"react-native": "0.61.1",
```

**Comment by davidskaarup** [comment]
> Recommendation: downgrade react-native
> 
> rewrite package.json
> 
> ```json
> "react-native": "0.61.1",
> ```
> 
> or
> react-native 0.61.2
> 
> add index.js
> 
> ```js
> import { gestureHandlerRootHOC } from 'react-native-gesture-handler';
> 
> AppRegistry.registerComponent(appName, () => gestureHandlerRootHOC(App));
> ```
> 
> there are no problem release builds

@YutaSugimura none of those solutions worked for me (iOS). Are there any way to fix the crashes in release mode for iOS?

**Comment by klosowsk** [comment]
Same here, it works for android but not for IOS.
Even when I do:
```
`import 'react-native-gesture-handler';`
... #### ...
import {gestureHandlerRootHOC} from 'react-native-gesture-handler';
AppRegistry.registerComponent(appName, () => gestureHandlerRootHOC(App));
```
```
"react": "16.9.0",
"react-native": "0.61.2",
"react-native-maps": "^0.26.1",
"react-native-reanimated": "^1.3.0",
"react-native-gesture-handler": "^1.4.1",
```
The YellowBox works, but it crashes the app in production, so it's not a valid solution, any updates?

**Comment by RigoTheDev** [comment]
The fact that we have to deal with a dumb bug like this is very annoying. How do we fix this?

**Comment by scarcoco** [comment]
> I try put
> 
> 1 - import 'react-native-gesture-handler' at the top of index.js
> 2 - YellowBox.ignoreWarnings([
> 'RCTRootView cancelTouches'
> ]);
> 
> And don't works. Any news about the solution?

YellowBox.ignoreWarnings works

**Comment by RigoTheDev** [comment]
@scarcoco Still crashes in release builds

**Comment by andrerfneves** [comment]
Have attempted the many 'react-native-gesture-handler'-related fixes that were discussed in this thread (and many others) and have not been able to make this work. This issue only crashes iOS release builds. iOS dev and Android (release and dev) work great. 

I'm happy to mute the issue using YellowBox, as long as the deprecation warning for RCTRootView cancelTouches doesn't break the app entirely. I've also tried downgrading React Native to 61.1 as mentioned above but that also didn't fix it. 

How can we help solve this issue? After reading source code and researching online I've yet to understand how the deprecation warning is causing the entire app to crash. https://github.com/facebook/react-native/commit/36307d87e1974aff1abac598da2fd11c4e8e23c1

**Comment by andrerfneves** [comment]
Seems that @kmagiera released 1.5.0 which fixes this issue: https://github.com/kmagiera/react-native-gesture-handler/releases/tag/1.5.0

Testing now, will report back.

**Comment by Kamran1122** [comment]
@andrerfneves after upgrading to 1.5.0 it seems not to show the warning in IOS dev mode anymore. Haven't tested in release mode yet.

**Comment by tattivitorino** [comment]
Same as @Kamran1122 here!

**Comment by roberthewitt** [comment]
After updating the dependency, I can confirm that dev builds no longer show the yellow-box. 
But this still causes a crash on iOS 13+ release builds

**Comment by ghoshabhi** [comment]
I have the latest (v1.5.2) release of the `react-native-gesture-handler` package:  https://github.com/kmagiera/react-native-gesture-handler/releases/tag/1.5.2, but still see the issue

**Comment by Rambatino** [comment]
@ghoshabhi I removed the import from my index.js, updated to 1.5.2 and STILL had that issue.

I then imported it in my index.js again and the warning went away.

So you have to do both.

**Comment by ryanmagoon** [comment]
in 1.5.3 the issue went away for me. I did have to restart the app/bundler for it to go away after updating

**Comment by stigi** [comment]
This was fixed with software-mansion/react-native-gesture-handler#898 and got released in v1.5.3. Hope everyone gets to upgrade fast. 🤞

**Comment by jsunwoo** [comment]
Thanks @stigi it works like a charm.
